### PR TITLE
[2.6] Bump rancher-cis-benchmark:2.0.5-rc4

### PR DESCRIPTION
**Pull request by dhruvmewada15**
This PR has changes for addition of cis-1.20 profiles for:

- cis-1.20 
- rke-profile-permissive-1.20
- rke-profile-hardened-1.20
- rke2-profile-permissive-1.20
- rke2-profile-hardened-1.20
- k3s-profile-permissive-1.20
- k3s-profile-hardened-1.20
- cis-1.23
- rke-profile-permissive-1.23
- rke-profile-hardened-1.23
- rke2-profile-permissive-1.23
- rke2-profile-hardened-1.23
- k3s-profile-permissive-1.23
- k3s-profile-hardened-1.23

In addition to above profiles for cis this PR has updated security-scan image which has chnages for below issues in security-scan
Linked Issue:
https://github.com/rancher/cis-operator/issues/135
https://github.com/rancher/cis-operator/issues/153
https://github.com/rancher/cis-operator/issues/156
https://github.com/rancher/cis-operator/issues/149
https://github.com/rancher/rancher/issues/37944

### CODE REVIEW

**Review by prachidamle — APPROVED** [review]

File: release.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ rancher-aks-operator-crd:
 rancher-csp-adapter:
 - 1.0.0+up1.0.0-rc2
 rancher-cis-benchmark:
-- 2.0.5-rc2
+- 2.0.5-rc3
 rancher-cis-benchmark-crd:
-- 2.0.5-rc2
+- 2.0.5-rc3
+system-upgrade-controller:
```
Comment (prachidamle) [inline release.yaml:25]: Why are we adding system-upgrade-controller in this PR?

**Comment by prachidamle** [comment]
Rest of the changes look good; except for the above comment.

File: release.yaml (hunk 1 above)
Comment (dhruvmewada15) [reply release.yaml]: Hi @prachidamle. I have not added it manually - it came as part of rebase from [upstream dev-v2.6](https://github.com/rancher/charts/blob/dev-v2.6/release.yaml#:~:text=system%2Dupgrade%2Dcontroller,%2D%20100.0.3%2Bup0.3.2) when I was trying to solve conflicts.

**Review by prachidamle — APPROVED** [review]

**Review by doflamingo721 — APPROVED** [review]
lgtm

**Pull request merged**